modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf

```diff
@@ -2,9 +2,6 @@ process GATK4_DETERMINEGERMLINECONTIGPLOIDY {
     tag "$meta.id"
     label 'process_single'
 
-    if(params.enable_conda){
-        error "Conda environments cannot be used for GATK4/DetermineGermlineContigPloidy at the moment. Please use docker or singularity containers."
```

Comment:
Don't know. It has to be tested but this will only affect a small minority of modules so we can add that afterwards. All of the modules I removed it from can be found in the commits I labelled `Remove error raise for modules that dont support conda` above.